pkg/file/builder_test.go

```diff
@@ -3853,3 +3870,151 @@ func Test_stateBuilder_expressionRoutes_kong370_withKonnect(t *testing.T) {
 		})
 	}
 }
+
+func Test_stateBuilder_ingestPluginEntities(t *testing.T) {
+	rand.Seed(42)
```

Comment:
This is deprecated as per: https://pkg.go.dev/math/rand#Seed

> Deprecated: As of Go 1.20 there is no reason to call Seed with a random value. Programs that call Seed with a known value to get a specific sequence of results should use New(NewSource(seed)) to obtain a local random generator.

Reply:
I have added changes based on the recommendations in rand package. We need that for deterministically generating uuids for the entities.